lcs/agents/acs2/ACS2.py

```diff
@@ -33,7 +33,7 @@ def _run_trial_explore(self, env, time, current_trial=None) \
         # Initial conditions
         steps = 0
         raw_state = env.reset()
-        state = self.cfg.environment_adapter.to_genotype(raw_state)
+        state = Perception(self.cfg.environment_adapter.to_genotype(raw_state))
```

Comment:
Do we need casting?

Reply:
That's the point of this commit. If we don't cast to Perception, we get a simple tuple, which doesn't look very well when printed with str()

Reply:
Or perhaps it would be better to do the casting in the default implementation of EnvironmentAdapter instead!

Reply:
Yep - that would be better.

Reply:
Ok, I reverted this for now, let's do it in a separate PR